src/main/java/org/web3j/protocol/parity/JsonRpc2_0Parity.java

```diff
@@ -9,207 +8,324 @@
                 ID,
                 web3jService,
-                EthSendTransaction.class);
+                BooleanResponse.class);
     }
 
     @Override
-    public Request<?, VoidResponse> personalSetAccountName(
-            String accountId, String newAccountName) {
+    public Request<?, BooleanResponse> paritySetNewDappsAddresses(
+            ArrayList<String> availableAccountIds) {
+        return new Request<>(
+                "parity_setNewDappsAddresses",
+                Arrays.asList(availableAccountIds),
+                ID,
+                web3jService,
+                BooleanResponse.class);
+    }
 
+    @Override
+    public Request<?, BooleanResponse> paritySetNewDappsDefaultAddress(String defaultAddress) {
         return new Request<>(
-                "personal_setAccountName",
-                Arrays.asList(accountId, newAccountName),
+                "parity_setNewDappsDefaultAddress",
+                Arrays.asList(defaultAddress),
                 ID,
                 web3jService,
-                VoidResponse.class);
+                BooleanResponse.class);
     }
 
     @Override
-    public Request<?, VoidResponse> personalSetAccountMeta(
-            String accountId, Map<String, Object> metadata) {
+    public Request<?, BooleanResponse> parityTestPassword(String accountId, String password) {
         return new Request<>(
-                "personal_setAccountMeta",
-                Arrays.asList(accountId, metadata),
+                "parity_testPassword",
+                Arrays.asList(accountId, password),
                 ID,
                 web3jService,
-                VoidResponse.class);
+                BooleanResponse.class);
     }
 
     @Override
-    public Request<?, PersonalAccountsInfo> personalAccountsInfo() {
+    public Request<?, PersonalSign> paritySignMessage(
+            String accountId, String password, String hexMessage) {
         return new Request<>(
-                "personal_accountsInfo",
-                Collections.<String>emptyList(),
+                "parity_signMessage",
+                Arrays.asList(accountId,password,hexMessage),
                 ID,
                 web3jService,
-                PersonalAccountsInfo.class);
+                PersonalSign.class);
     }
 
     @Override
-    public Request<?, PersonalRequestsToConfirm> personalRequestsToConfirm() {
+    public Request<?, SignerRequestsToConfirm> signerRequestsToConfirm() {
         return new Request<>(
-                "personal_requestsToConfirm",
+                "signer_requestsToConfirm",
                 Collections.<String>emptyList(),
                 ID,
                 web3jService,
-                PersonalRequestsToConfirm.class);
+                SignerRequestsToConfirm.class);
     }
 
     @Override
-    public Request<?, EthSendTransaction> personalConfirmRequest(
+    public Request<?, EthSendTransaction> signerConfirmRequest(
```

Comment:
On second thoughts - you can get rid of the signer module methods - given its only a subset of them anyway. I'd question how useful they are for web3j users.